spec/plaid_spec.rb

```diff
@@ -122,4 +122,14 @@
       end
     end
   end
+
+  describe Plaid::Info do
```

Comment:
i have it on my machine, not sure how i got it, but i got it sometime in december. looks like it's meant to supply data to the plaid_ruby config (normally it comes from the consuming app)

contents:
client_id: test_id
secret: test_secret
base_url: https://tartan.plaid.com/